pom.xml

```diff
@@ -94,6 +93,12 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.21</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Otherwise SLF4J will whine about no implementation being bound at test runtime and all log messages during tests will be dropped by [`NOPLogger`](http://www.slf4j.org/api/org/slf4j/helpers/NOPLogger.html).